examples/performance-api/src/pipeline_entry/index.tsx

```diff
@@ -3,9 +3,21 @@ import { root, useMemo, useState } from "@lynx-js/react";
 import type { PerformanceEntry, PipelineEntry } from "@lynx-js/types";
 import "./index.scss";
 
+export const RenderPropItem = (props: { value: string; hightLightProp?: string }) => {
```

Comment:
I don't think export `RenderPropItem` from this file is correct.

This would result in `root.render` being called twice.

Please put it in another file.